src/components/Scan-Button.tsx

```diff
@@ -0,0 +1,11 @@
+import { Link } from 'react-router-dom';
+
+export const InfoButton = () => {
```

Comment:
This component's name doesn't match the file name

Reply:
Done sir @stephane-segning